dist/test_data.py

```diff
@@ -153,7 +155,10 @@ def __ge__(self, other):
 #
 # Configuration that applies to the workload_generator component.
 #
-workload_generator = enabled_config_true + component_config + populate_config + [
+workload_generator = enabled_config_true + component_config + [
+    Config('populate_config', '', r'''
```

Comment:
What is the added value to redefine a Config(...) instead of using directly the define populate_config ?

Reply:
I personally think its a bit less confusing in the config_poc_test_stress.txt.